pkg/config/justification_config.go

```diff
@@ -148,5 +152,12 @@ func (cfg *JustificationConfig) ToFlags(set *cli.FlagSet) *cli.FlagSet {
 		Usage:   "The maximum TTL that a token can have.",
 	})
 
+	f.StringSliceVar(&cli.StringSliceVar{
```

Comment:
For each of the plugin, multiple things are needed:
1. name 
2. checksum
3. plugin path
4. Other EnvVars/flags needed by the specific plugin

1-3 are the same for all the plugin, so it is easy to pass back. I am not clear how to inject 4 since those can be different for each plugin. Would it also be a JSON we need to unpack to inject all the EnvVars/flags?